tests/test_index_utils.py

```diff
@@ -291,19 +266,7 @@ def test_null_value_should_not_be_index(config):
         'city': ''
     }
     index_document(doc)
-    assert 'city' not in DB.hgetall('d|xxxx')
-
-
-def test_field_with_only_non_alphanumeric_chars_is_not_indexed():
-    doc = {
-        'id': 'xxxx',
-        'lat': '49.32545',
-        'lon': '4.2565',
-        'name': 'Lilas',
-        'city': '//'
-    }
-    index_document(doc)
-    assert 'city' not in DB.hgetall('d|xxxx')
```

Comment:
This check was dummy: `'city'` is not in the retrieved document, because keys are bytes, not str.
So the correct test would have been `b'city' not in…`, and of course it's not passing that way…